modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogHeader.ts

```diff
@@ -52,7 +52,7 @@ const renderTitle = (
 
   return {
     dom: {
-      tag: 'div',
+      tag: 'h1',
```

Comment:
have you checked this has no impact on our skins? there are more skins than just the default 🤔 

maybe make a special note for QA to check that.

Reply:
Will do that and make a note, the styles for the heading title use class selector.